### ISSUE TITLE
Differences between framed vs oriented

### ISSUE THREAD

**Issue by calintat** [homotopy-io/homotopy-rs#616]
Reidemeister I move (wire needs to be oriented and invertible):

![homotopy_io_export](https://user-images.githubusercontent.com/16837293/180812161-8be75cec-1c49-4b52-9711-21ee96787e86.svg) ![homotopy_io_export(1)](https://user-images.githubusercontent.com/16837293/180812183-9b02a4ae-2fde-44bb-80db-6e1ca43d39d1.svg)

**Comment by calintat** [comment]
Any invertible monoid gives rise to a Frobenius-like structure (monoid needs to be oriented, wire may be framed):

![homotopy_io_export(4)](https://user-images.githubusercontent.com/16837293/180814850-0182c03c-4308-497d-a5e9-7e3560ec7958.svg) ![homotopy_io_export(5)](https://user-images.githubusercontent.com/16837293/180814859-4c6576f9-e687-4da4-aac8-1fc7f7153962.svg)

**Comment by NickHu** [comment]
> Any invertible monoid gives rise to a Frobenius-like structure (monoid needs to be oriented, wire may be framed):
> 
> ![homotopy_io_export(4)](https://user-images.githubusercontent.com/16837293/180814850-0182c03c-4308-497d-a5e9-7e3560ec7958.svg) ![homotopy_io_export(5)](https://user-images.githubusercontent.com/16837293/180814859-4c6576f9-e687-4da4-aac8-1fc7f7153962.svg)

In what sense is this Frobenius? The connectivity between the two wires has been lost

**Comment by calintat** [comment]
> > Any invertible monoid gives rise to a Frobenius-like structure (monoid needs to be oriented, wire may be framed):
> > ![homotopy_io_export(4)](https://user-images.githubusercontent.com/16837293/180814850-0182c03c-4308-497d-a5e9-7e3560ec7958.svg) ![homotopy_io_export(5)](https://user-images.githubusercontent.com/16837293/180814859-4c6576f9-e687-4da4-aac8-1fc7f7153962.svg)
> 
> In what sense is this Frobenius? The connectivity between the two wires has been lost

Yeah I probably shouldn't have called it Frobenius, though it's like a degenerate Frobenious law.

**Comment by calintat** [comment]
Here is another interesting one...

Given an oriented 2-cell on a (framed) invertible 1-cell, its trace contracts to a scalar version of the 2-cell.
![homotopy_io_export](https://user-images.githubusercontent.com/16837293/182621307-5fb5ea1b-de55-472e-a31c-e3235a0aa83b.svg) ![homotopy_io_export(1)](https://user-images.githubusercontent.com/16837293/182621317-42d9b60b-238f-44c3-b84b-56be077ce858.svg)

**Comment by doctorn** [comment]
> Here is another interesting one...
> 
> Given an oriented 2-cell on a (framed) invertible 1-cell, its trace contracts to a scalar version of the 2-cell. ![homotopy_io_export](https://user-images.githubusercontent.com/16837293/182621307-5fb5ea1b-de55-472e-a31c-e3235a0aa83b.svg) ![homotopy_io_export(1)](https://user-images.githubusercontent.com/16837293/182621317-42d9b60b-238f-44c3-b84b-56be077ce858.svg)

This is a special case of the contractibility of disks right?

**Comment by doctorn** [comment]
Fun!

![Contract](https://user-images.githubusercontent.com/6421560/182643234-9b3f492c-493d-4234-bba1-9b6ec53b68fd.gif)

**Comment by jamievicary** [comment]
Thanks for these great points on this thread. Manuel and I had a long discussion about it. Let's all talk together when you guys are back from FLOC.
